echo360/main.py

```diff
@@ -137,6 +137,17 @@ def handle_args():
                               institution's SSO; i.e. it will disable the automatic \
                               redirection which is the default behavior.",
     )
+    parser.add_argument(
+        "--two-feeds",
```

Comment:
I think calling it as `alternative_feeds` would be more obvious of what it is doing, i.e. the flag could be `--alternative-feeds`, `-a`, and later on in the code you can set variables as `self.download_alternative_feeds = alternative_feeds`, as well as other instance of the flag.

You will need to remove the short flags from
```python
    parser.add_argument(
        "--after-date",
        "-a",
        dest="after_date",
        help="Only download lectures newer than AFTER_DATE \
                             (inclusive). Note: this may be combined with \
                             --before-date.",
        metavar="AFTER_DATE(YYYY-MM-DD)",
    )
    parser.add_argument(
        "--before-date",
        "-b",
        dest="before_date",
        help="Only download lectures older than BEFORE_DATE \
                              (inclusive). Note: this may be combined with \
                              --after-date",
        metavar="BEFORE_DATE(YYYY-MM-DD)",
    )
```
to
```python
    parser.add_argument(
        "--after-date",
        dest="after_date",
        help="Only download lectures newer than AFTER_DATE \
                             (inclusive). Note: this may be combined with \
                             --before-date.",
        metavar="AFTER_DATE(YYYY-MM-DD)",
    )
    parser.add_argument(
        "--before-date",
        dest="before_date",
        help="Only download lectures older than BEFORE_DATE \
                              (inclusive). Note: this may be combined with \
                              --after-date",
        metavar="BEFORE_DATE(YYYY-MM-DD)",
    )
```
as a breaking change, but I don't think much people are using those flags so that should be fine.